### PR TITLE
Submit project proposals list for GSoC 2021

### PR DESCRIPTION
**Pull request by LinuxSuRen**
Currently, we have three project proposals. Please help to review these ideas

@benjaminhuo @wanjunlei @shaowenchen @FeynmanZhou 

@kubesphere/sig-advocacy-and-outreach

### CODE REVIEW

**Comment by LinuxSuRen** [comment]
close #280

File: sig-advocacy-and-outreach/gsoc/README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
-KubeSphere in [Google Summber of Code (AKA GSoC)](https://summerofcode.withgoogle.com/). You can find our propsosal in this page. Or you also can submit your proposal to KubeSphere community.
+KubeSphere in [Google Summber of Code (aka GSoC)](https://summerofcode.withgoogle.com/). You can find our propsosals in this page. Or you also can submit your proposal to KubeSphere community.
```
Comment (FeynmanZhou) [inline sig-advocacy-and-outreach/gsoc/README.md:1]: propsosals =》 proposals

File: sig-advocacy-and-outreach/gsoc/README.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
-KubeSphere in [Google Summber of Code (AKA GSoC)](https://summerofcode.withgoogle.com/). You can find our propsosal in this page. Or you also can submit your proposal to KubeSphere community.
+KubeSphere in [Google Summber of Code (aka GSoC)](https://summerofcode.withgoogle.com/). You can find our proposals in this page. Or you also can submit your proposal to KubeSphere community.
 
 ## Proposals
 
-TODO
+| Project | Area | Skills to study/improve |
+| --- | --- | --- |
+| [KubeSphere Jenkins Client Refactor](kubeSphere-jenkins-client-refactor.md) <br/>Split the Jenkins client from KubeSphere. Or use an existing Jenkins client. </br>Potential Mentor(s): [Rick](https://github.com/LinuxSuRen/), [Shaowen Chen](https://github.com/shaowenchen/)| DevOps | Golang, REST API/OpenAPI, Jenkins, Kubernetes |
+| [KubeSphere Tekton Integration](kubeSphere-tekton-integration.md) <br/>Integrate Tekton as an alternative CI/CD engine of KubeSphere DevOps. <br/>Potential Mentor(s): [Rick](https://github.com/LinuxSuRen/), [Shaowen Chen](https://github.com/shaowenchen/)| DevOps | Golang, Tekton, Jenkins, Kubernetes |
```
Comment (FeynmanZhou) [inline sig-advocacy-and-outreach/gsoc/README.md:8]: @Felixnoo Please help to refine these two proposals: KubeSphere Jenkins Client Refactor and KubeSphere Tekton Integration, you would better communicate with @LinuxSuRen to make it much clear.

File: sig-advocacy-and-outreach/gsoc/README.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
...
+| [Customize receiving users via webhook](customize-receiving-users-via-webhook.md) <br/>Make the notification manager support to get a user list that will receive the notifications from a webhook. <br/> Potential Mentor(s): [benjaminhuo](https://github.com/benjaminhuo), [lei](https://github.com/wanjunlei)| Notification Manager | Golang, REST API, Kubernetes |
+| [Exposure of LoadBalancer service using iptables nat rules](exposure-of-loadBalancer-service-using-iptables-nat-rules.md) <br/>Multiplex the node ip address as the external ip address of the LoadBalancer service, and forward the traffic to the cluster ip of the service through the iptables nat rule. <br/>Potential Mentor(s): [Zhengyi Lai](https://github.com/zheng1), [Jiong Duan](https://github.com/duanjiong)| Network | Golang, Kubernetes, Linux network(iptables、ipvs) |
```
Comment (FeynmanZhou) [inline sig-advocacy-and-outreach/gsoc/README.md:10]: @Patrick-LuoYu Please help to refine these two proposals: Customize receiving users via webhook and Exposure of LoadBalancer service using iptables nat rules. You would better communicate with @wanjunlei, @benjaminhuo and @duanjiong @zheng1 to make them much clear.

**Review by FeynmanZhou — APPROVED** [review]

**Pull request merged**